app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -50,18 +51,19 @@ define([
       }
 
       return this._fetchProfileImage(account)
-        .then(function (result) {
-          if (result && result.avatar) {
-            self.$(wrapperClass).append(new Image());
-            self.$(wrapperClass + ' img').attr('src', result.avatar);
+        .then(function (img) {
+          if (img) {
+            self.$(wrapperClass).append(img);
             self.$(wrapperClass).removeClass('with-default');
+            self.logEvent(self.className + '.profile_image_shown');
           } else {
-            self.$(wrapperClass).addClass('with-default');
+            self.logEvent(self.className + '.profile_image_not_shown');
           }
-          return result;
-        }, function () {
+        })
+        .then(null, function () {
           // Ignore errors; the default image will be shown.
           self.$(wrapperClass).addClass('with-default');
+          self.logEvent(self.className + '.profile_image_not_shown');
```

Comment:
Can you document these new errors in https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md?